### ISSUE TITLE
UI: Dock Recover Button

### ISSUE THREAD

**Issue by CaptainLame** [MaFi-Games/Captain-of-Industry-issues#279]
The "Shovel ground to recover ocean access | 10 U" button spans the full width of the UI, making it look like a banner instead of a button. The width should be reduced if possible (perhaps made dynamic width based on language?) to make it look like a clickable element.

The wording could also possibly be adjusted to make it more clear that it is a button to click, not instruction to the player, perhaps "Recover Ocean Access".

**Comment by Slayman** [comment]
Did the button actually do anything for you? I have a "similar" issue but the button is in a bad position in the frame of the cargo dock and clicking it does nothing.

**Comment by CaptainLame** [comment]
I noted this while helping another player troubleshoot; I haven't had a situation where I needed this button due to dumping over the dock. The player I was helping was unaware that it was a button and didn't even try to click it.

I did get the button to pop when I placed a planned 6-slot dock slightly too close to another dock and it somehow caused a minor terrain change that flagged the _second_ module of that dock as blocked. The button did nothing in that case.
